etc/smartdns/smartdns.conf

```diff
@@ -206,6 +206,8 @@ log-level info
 # ipset /www.example.com/block, set ipset with ipset name of block 
 # ipset /www.example.com/-, ignore this domain
 
+# nftset [no]
```

Comment:
Consider adding a configuration similar to the following:

```shell
nftset /domain/[setname]
ntfset-timeout [yes|no]
```

Reply:
for nft, we hv to specify three things: table type, table name and set name. should we consider a better config grammar for it?

Reply:
like this?
nftset /example.com/4#ip#test#setv4
nftset /example.com/6#ip#test#setv6

Reply:
lgtm, will follow this.